908117.pdf

EliCili opened the pull request:
Salve, per poter rispondere alle mie precedenti domande, il software che uso è Revit.
Grazie della disponibilità e buona serata. 
Elisa Ciliberti

strumet commented:
Ciao Caterina,
i file sono nominati correttamente. Dovresti però rimuovere il file _[MO-D_2_STUDIO-SEZIONE.pdf](https://github.com/caterinaperego/strumet.github.io/blob/e046d255b826e19a11de8c0d9854cd8861f16560/ex/vectors/MO-D_2_STUDIO-SEZIONE.pdf)_: non è infatti prevista la sua presenza in questa cartella.
Grazie,
mf

caterinaperego commented:
Buongiorno, dovrei aver sistemato. 
Grazie Caterina Perego

> Il giorno 03 apr 2018, alle ore 10:30, strumet <notifications@github.com> ha scritto:
> 
> Ciao Caterina,
> i file sono nominati correttamente. Dovresti però rimuovere il file MO-D_2_STUDIO-SEZIONE.pdf <https://github.com/caterinaperego/strumet.github.io/blob/e046d255b826e19a11de8c0d9854cd8861f16560/ex/vectors/MO-D_2_STUDIO-SEZIONE.pdf>: non è infatti prevista la sua presenza in questa cartella.
> Grazie,
> mf
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/strumet/strumet.github.io/pull/48#issuecomment-378171130>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AjXg-ZwPjPzf1pVT5Sx92aLElwjEeTu4ks5tkzMigaJpZM4TEnB2>.
>

strumet commented:
Perfetto :+1:

EliCili committed:
908117.pdf

Salve, per poter rispondere alle mie precedenti domande, il software che uso è Revit.
Grazie della disponibilità e buona serata. 
Elisa Ciliberti

strumet commented:
Grazie Elisa.
Riprendo dunque le domande fatte nel [precedente pull request](https://github.com/strumet/strumet.github.io/pull/47):

- _Come si cambia la linea di sezione da continua a tratto-punto?_

Su revit la modifica dello stile di rappresentazione di un oggetto può essere effettuata:

- limitatamente ad una singola vista tramite le proprietà _Visibility/Graphics Overrides_ relative ad essa, oppure
- estesa a tutte le rappresentazioni del progetto tramite le proprietà definite in _Manage > Object Styles_

Se vuoi cambiare lo stile della linea di sezione dovrai usare quest'ultima opzione, spostarti nel tab _Annotation Objects_ ed impostare il tipo di linea che desidera assegnare alle linee di sezione nella riga ad essa dedicata (_Section Line_)

![Screenshot from 2019-04-15 10-13-38](https://user-images.githubusercontent.com/6952124/56119335-3ada4300-5f6c-11e9-97da-108b1b254795.png)

- _Come posso modificare il layout della freccia che mi indica la direzione verso la quale si guarda la sezione?_

Su Revit tutte le entità, sia quelle reali che quelle grafiche, sono definite attraverso le famiglie.
Anche le linea di sezione fa uso di famiglie: per assegnare il layout di freccia desiderata dovrai quindi selezionare la linea di sezione e modificare il valore assegnato al parametro "Section Tag" dalla finestra di modifica del tipo (comando _Edit Type_).

![Screenshot from 2019-04-15 10-38-36](https://user-images.githubusercontent.com/6952124/56119391-55142100-5f6c-11e9-8322-d28d74b80f92.png)

Se le librerie di cui disponi non contengono una freccia che ti soddisfa puoi crearne una personalizzata utilizzando il template _Annotations > Metric Section Head_ presente nella libreria di default di Revit.

In un prossimo messaggio provvederò ad effettuare una revisione della tavola da te prodotta.
mf

strumet commented:
Ciao Elisa,
la tavola realizzata per l'attività 2 risulta essere **rasterizzata**. Questo va **sempre evitato** perché comporta una riduzione della qualità della tavola ed un inutile incremento del peso del file.

Di seguito troverai le mie note di revisione sul lavoro.
Saluti,
mf

![908117](https://user-images.githubusercontent.com/6952124/56125621-a1199280-5f79-11e9-937f-f57225991054.png)

EliCili commented:
Buongiorno
Grazie mille delle risposte alle domande e delle correzioni, provvederò a correggere e reinviare il file.  Elisa Ciliberti

Il 15 apr 2019 12:30 PM, strumet <notifications@github.com> ha scritto:

Ciao Elisa,
la tavola realizzata per l'attività 2 risulta essere rasterizzata. Questo va sempre evitato perché comporta una riduzione della qualità della tavola ed un inutile incremento del peso del file.

Di seguito troverai le mie note di revisione sul lavoro.
Saluti,
mf

[908117]<https://user-images.githubusercontent.com/6952124/56125621-a1199280-5f79-11e9-937f-f57225991054.png>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/strumet/strumet.github.io/pull/48#issuecomment-483197921>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AuLU6IEAR5Y0_h-K0VK8cNQIx92SBO_mks5vhFTVgaJpZM4cuK4p>.
